src/main/java/org/jfrog/hudson/pipeline/declarative/steps/generic/GenericStep.java

```diff
@@ -34,8 +34,7 @@
     private String specPath;
     boolean failNoOp;
 
-    @DataBoundConstructor
```

Comment:
Was this annotation removed intentionally?

Reply:
Yes, these two steps are never called directly